<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BAON</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/graphics/app_icon.png</normaloff>:/graphics/app_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layBasePath">
     <item>
      <widget class="QLabel" name="lblBasePath">
       <property name="text">
        <string>Base Path</string>
       </property>
       <property name="buddy">
        <cstring>txtBasePath</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtBasePath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowse">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbxOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="chkScanRecursive">
        <property name="text">
         <string>Recursively scan subfolders</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkUsePath">
        <property name="text">
         <string>Use path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkUseExtension">
        <property name="text">
         <string>Use extension</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbxRules">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>112</height>
      </size>
     </property>
     <property name="title">
      <string>Rename Rules</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTextEdit" name="txtRules">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbxFiles">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Renamed Files</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="RenameTable" name="tblFiles">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbxStatus">
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblStatus">
        <property name="text">
         <string>Enter the base path for the files that are to be renamed.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenameTable</class>
   <extends>QTableWidget</extends>
   <header>gui.RenameTable</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MainWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MainWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkScanRecursive</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>_onDataEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUsePath</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>_onDataEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUseExtension</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>_onDataEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtBasePath</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>_onDataTyped()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtBasePath</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>_onDataEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtBasePath</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>_onDataEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBrowse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>_onClickedBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>785</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtRules</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>_onDataTyped()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_onDataEdited()</slot>
  <slot>_onDataTyped()</slot>
  <slot>_onClickedBrowse()</slot>
 </slots>
</ui>
